File: PROG2/rectangulo.java
public class rectangulo {
    public float base;
    public float altura;
    public rectangulo(float a , float b ){
        base = a;
        altura = b;
    }
public float perimetro(){
        return 2 *(base + altura);
}
public float area(){
        return base * altura;
}

    public float getBase() {
        return base;
    }

    public void setBase(float base) {
        this.base = base;
    }

    public float getAltura() {
        return altura;
    }

    public void setAltura(float altura) {
        this.altura = altura;
    }

    public static void main(String[] args) {
        rectangulo r1 = new rectangulo(2,3);
        System.out.println("perimetro="+ r1.perimetro());
        System.out.println("area="+ r1.area());
    }
}

File: Rectangulo.py
class Rectangulo:
    def __init__(self ,a,b):
        self.base = a
        self.altura =b
    def perimetro (self):
        return 2 * (self.base + self.altura)
    def area (self):
        return self.base *  self.altura
r1 =Rectangulo(2,3)
print("perimetro=", r1.perimetro())
print("area=", r1.area())